docs/reference/enumeration-classes.txt

```diff
@@ -4,190 +4,20 @@ Enumeration Classes
-
-Definition
-~~~~~~~~~~
-
-.. phpclass:: MongoDB\\Model\\DatabaseInfoIterator
-
-   This interface extends PHP's :php:`Iterator <manual/en/class.iterator.php>`
-   interface. An instance of this interface is returned by
-   :phpmethod:`MongoDB\\Client::listDatabases()`.
-
-Methods
-~~~~~~~
-
-This interface adds no new methods to :php:`Iterator
-<manual/en/class.iterator.php>`, but specifies that :php:`current()
-<manual/en/iterator.current.php>` will return an instance of
-:phpclass:`MongoDB\\Model\\DatabaseInfo`.
-
-----
-
-MongoDB\\Model\\IndexInfo
--------------------------
-
-.. phpclass:: MongoDB\\Model\\IndexInfo
-
-   This class models information about an index. Instances of this class are
-   returned by traversing a :phpclass:`MongoDB\\Model\\IndexInfoIterator`,
-   which is returned by :phpmethod:`MongoDB\\Collection::listIndexes()`.
-
-   This class implements PHP's :php:`ArrayAccess <arrayaccess>` interface. This
-   provides a mechanism for accessing index fields for which there exists no
-   helper method. :php:`isset() <isset>` may be used to check for the existence
-   of a field before accessing it with ``[]``.
-
-   .. note::
-
-      The :phpclass:`MongoDB\\Model\\IndexInfo` class is immutable. Attempting
-      to modify it via the :php:`ArrayAccess <arrayaccess>` interface will
-      result in a :phpclass:`MongoDB\\Exception\\BadMethodCallException`.
-
-Methods
-~~~~~~~
-
-.. toctree::
-   :titlesonly:
-
-   /reference/method/MongoDBModelIndexInfo-getKey
-   /reference/method/MongoDBModelIndexInfo-getName
-   /reference/method/MongoDBModelIndexInfo-getNamespace
-   /reference/method/MongoDBModelIndexInfo-getVersion
-   /reference/method/MongoDBModelIndexInfo-is2dSphere
-   /reference/method/MongoDBModelIndexInfo-isGeoHaystack
-   /reference/method/MongoDBModelIndexInfo-isSparse
-   /reference/method/MongoDBModelIndexInfo-isText
-   /reference/method/MongoDBModelIndexInfo-isTtl
-   /reference/method/MongoDBModelIndexInfo-isUnique
-
-----
-
-MongoDB\\Model\\IndexInfoIterator
----------------------------------
-
-Definition
-~~~~~~~~~~
-
-.. phpclass:: MongoDB\\Model\\IndexInfoIterator
-
-   This interface extends PHP's :php:`Iterator <manual/en/class.iterator.php>`
-   interface. An instance of this interface is returned by
-   :phpmethod:`MongoDB\\Collection::listIndexes()`.
+   /reference/class/MongoDBModelCollectionInfo
+   /reference/class/MongoDBModelCollectionInfoIterator
+   /reference/class/MongoDBModelDatabaseInfo
+   /reference/class/MongoDBModelDatabaseInfoIterator
+   /reference/class/MongoDBModelIndexInfo
+   /reference/class/MongoDBModelIndexInfoIterator
 
-Methods
-~~~~~~~
+- :ref:`CollectionInfo <php-model-collection-info>`
+- :ref:`CollectionInfoIterator <php-model-collection-info-iterator>`
+- :ref:`DatabaseInfo <php-model-database-info>`
+- :ref:`DatabaseInfoIterator <php-model-database-info-iterator>`
+- :ref:`IndexInfo <php-model-index-info>`
+- :ref:`IndexInfoIterator <php-model-index-info-iterator>`
```

Comment:
Since the "Enumeration Classes" drawer is now clickable and expands (per https://github.com/mongodb/mongo-php-library/pull/1229/files#r1484800559), is this content even viewable anymore? I found no way to navigate to this page directly in the current staging deployment.

Reply:
It's not viewable right now (other than by editing the URL or using the right/left arrow buttons at the bottom of each page). I can make it viewable, or leave as is and remove the content on this page